components/.browserslistrc

```diff
@@ -1,6 +1,4 @@
-ie >= 11
-edge >= 13
-firefox >= 5
-opera >= 12
-safari >= 5
-chrome >= 56
+> 1%
```

Comment:
Recommended latest browserlist config https://github.com/vuejs/vue-cli/pull/5233

Reply:
We have to define the minimum supported browser versions.

Reply:
>1% is [all versions with >1% of the audience worldwide.](https://browsersl.ist/#q=%3E+1%25)
We can further specify versions, but needs to be updated manually. Let's discuss

From [browserlist tool](https://browsersl.ist)
![image](https://user-images.githubusercontent.com/12880059/210705537-24e28a95-51a3-4876-a081-a17fa0c8aac2.png)